index.html

```diff
@@ -21,7 +21,7 @@
 				</section>
 				<section><h3>What are intra-doc links?</h3>
 					<p>A way to link to items by name, not URL</p>
-					<pre><code class="language-rust">/// Link to [f()]
+					<pre><code class="language-rust">/// Link to [`f()`]
```

Comment:
Hmm, so I know the original blog post had these backticks, but I actually left them out intentionally so it would be more clear that they're optional. I've seen people confused about what the backticks do.

That said, this does make the links look prettier so I'm fine either way.

Reply:
seems fair!